json-mode-social-determinants-of-health/SDOH-Json-mode.ipynb

```diff
@@ -0,0 +1,593 @@
+       "\n",
+       "Educationally, the patient completed high school but hasn't pursued further studies, citing lack of funds and the immediate need to support their family after graduation. They rely on buses and trains for transportation, which complicates timely access to healthcare, often causing delays or missed appointments. Socially, the patient admits to feeling isolated, with most of their family living out of state after their divorce and a personal life that has been 'on hold' due to work and financial pressures. They have a basic health insurance plan with high co-payments, which has led to skipping some recommended medical tests and treatments.\n",
+       "\n",
+       "**Review of Systems:** Denies chest pain, shortness of breath, or fever. Reports occasional headaches.\n",
+       "\n",
+       "**Physical Examination:**\n",
+       "- General: Appears tired but is alert and oriented.\n",
+       "- Vitals: BP 142/89, HR 78, Temp 98.6°F, Resp 16/min\n",
+       "\n",
+       "**Assessment/Plan:**\n",
+       "- Continue to monitor blood pressure and diabetes control.\n",
+       "- Discuss affordable medication options with a pharmacist.\n",
+       "- Refer to a social worker to address food security, housing concerns, and access to healthcare services.\n",
+       "- Encourage the patient to engage with community support groups for social support.\n",
+       "- Schedule a follow-up appointment in 4 weeks or sooner if symptoms worsen.\n",
+       "\n",
+       "**Comments:** The patient's health concerns are compounded by socioeconomic factors, including employment status, housing stability, food security, and access to healthcare. Addressing these social determinants of health is crucial for improving the patient's overall well-being.\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Define the directory path\n",
+    "folder_path = 'clinical_notes/'\n",
+    "\n",
+    "# List all files in the directory\n",
+    "file_list = os.listdir(folder_path)\n",
+    "text_files = sorted([file for file in file_list if file.endswith('.txt')])\n",
+    "\n",
+    "with open(os.path.join(folder_path, text_files[0]), 'r') as file:\n",
+    "    clinical_note = file.read()\n",
+    "\n",
+    "display(Markdown(clinical_note))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "589d5075-8ffe-4fae-ac15-db8578229209",
+   "metadata": {},
+   "source": [
+    "### Define system and user prompts"
```

Comment:
```suggestion
    "### Define System and User Prompts"
```

Reply:
not particular about this, just saw that you capitalized other headers, so trying to keep it consistent! 😁